lib/project.rb

```diff
@@ -76,6 +76,7 @@ def init_class_variables
       @engine_type = Engine.select(@engine_name)
       @engine_tag = @engine_type.tag(@options)
       @engine_platform = @engine_type.platform(@logger, @options)
+      @engine_platform['vbs_state'] = @options.test.enable_vbs
```

Comment:
Should be done in HCKTest engine, as this does not related to Install engine for now